src/body/to_bytes.rs

```diff
@@ -13,6 +13,28 @@ use super::HttpBody;
 /// Care needs to be taken if the remote is untrusted. The function doesn't implement any length
 /// checks and an malicious peer might make it consume arbitrary amounts of memory. Checking the
 /// `Content-Length` is a possibility, but it is not strictly mandated to be present.
+///
+/// Create an empty `Body` stream.
+///
+/// # Example
+///
+/// ```
+/// let response = client.request(req).await?;
+///
+/// const MAX_ALLOWED_RESPONSE_SIZE: u32 = 1024;
+///
+/// let response_content_length: u32 = response.headers()
+///     .get(hyper::header::CONTENT_LENGTH)
+///     .expect("Failed to extract the CONTENT_LENGTH")
+///     .to_str().expect("Failed to_str() the CONTENT_LENGTH")
+///     .parse::<u32>().expect("Failed to parse the CONTENT_LENGTH");
+///
+/// if response_content_length < MAX_ALLOWED_RESPONSE_SIZE {
+///     let body_bytes = hyper::body::to_bytes(response.into_body()).await.expect("body::to_bytes failed");
```

Comment:
Since using `?` already, it'd be less noisy to keep using it and not need to have large `.expect(reason)`.